frontend/src/components/Layouts/Header/Components/BurgerMenu/BurgerMenu.module.scss

```diff
@@ -0,0 +1,116 @@
+@import '../../../../../variables';
+@import '../../../../../mixins';
+
+.burgerMenu {
+  display: none;
+  z-index: 1;
+  cursor: pointer;
+}
+.burgerHeader {
+  display: flex;
+  padding: 8px 16px 16px;
+  background-color: $white-color;
+  width: 100%;
+}
+.burgerIcon {
+  margin: 8px auto 0 0;
+}
+.actionIcons {
+  display: flex;
+}
+.search,
+.bouquet,
+.account,
+.cart {
+  display: flex;
+  padding: 8px 8px 0;
+}
+.openMenu {
```

Comment:
It has no position, when I open the menu, it should be on top of the content, not push it down. Add position:fixed